### PR TITLE
Customization Doc Updates

### PR DESCRIPTION
**Pull request by vasilvas99**

### CODE REVIEW

File: content/en/docs/build/dev-and-maintenance/rust/kanto-auto-deployer.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+The compiled binary takes a path to a directory containing the json manifests, parses them into Rust structures and sends gRPC requests to kanto container management to deploy these containers.
+If the container is already deployed the manifest is ignored.
+
+## Manifest structure
+
+Kanto auto deployer uses the exact same structure for its manifests as the internal representation of container state in kanto container management. For example:
+
+```json
+{
```
Comment (stlachev) [inline content/en/docs/build/dev-and-maintenance/rust/kanto-auto-deployer.md:23]: This is an old example. No ports
https://github.com/eclipse-leda/meta-leda/pull/106/commits/4e43671d8cc5881b9f0e6d962f79b4412d8f61b8#diff-02e135cb4b250fa858c9b2126603111b123cd7bc45160fa470ecbdf368885fec

Comment (vasilvas99) [reply content/en/docs/build/dev-and-maintenance/rust/kanto-auto-deployer.md]: Updated example manifest

File: content/en/docs/build/dev-and-maintenance/shell/sdv-health.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+```bash
+	KANTO_CM_LIST=$(${KANTO_CMD} list)
+```
+
+Output:
+
+```shell
+
+ID                                   	|Name                                 	|Image                                                       	|Status    	|Finished At                     |Exit Code 	|
```
Comment (stlachev) [inline content/en/docs/build/dev-and-maintenance/shell/sdv-health.md:61]: ID                                      |Name                                   |Image                                                                                                        |Status          |Finished At                     |Exit Code     |
-------------------------------------   |-------------------------------------  |------------------------------------------------------------                                                 |----------      |------------------------------  |----------    |
410dbf60-0802-4d5b-a2ba-4f1da802fcfc    |seatservice-example                    |ghcr.io/eclipse/kuksa.val.services/seat_service:v0.1.0                                                       |Running         |2023-01-31T11:14:06.125135195Z  |0             |
51b39dd3-0adf-4496-a496-0dc038e34643    |vum                                    |ghcr.io/eclipse-leda/leda-contrib-vehicle-update-manager/vehicleupdatemanager:main-1d8dca55a755c4b3c7bc06eabfa06ad49e068a48   |Running        |2023-01-31T11:14:06.268739879Z  |0             |
7ea315a5-4297-46e5-b5e5-5549f219d00d    |hvacservice-example                    |ghcr.io/eclipse/kuksa.val.services/hvac_service:v0.1.0                                                       |Running         |2023-01-31T11:14:06.405206125Z  |0             |
b29ae665-3881-496f-b8ab-51f1538e5fee    |feedercan                              |ghcr.io/eclipse/kuksa.val.feeders/dbc2val:v0.1.1                                                             |Running         |2023-01-31T11:14:06.514210945Z  |0             |
f09d30e3-ff2c-43df-91a3-c218d36f4bdc    |sua                                    |ghcr.io/eclipse-leda/leda-contrib-self-update-agent/self-update-agent:build-20                               |Running         |2023-01-31T11:14:06.623567528Z  |0             |
04e664b3-47a5-4983-9509-9d789c1d93b5    |databroker                             |ghcr.io/eclipse/kuksa.val/databroker:0.2.5                                                                   |Running         |2023-01-31T11:14:06.737905907Z  |0             |
37180f7e-9287-4106-af74-8651eed69020    |cloudconnector                         |ghcr.io/eclipse-leda/leda-contrib-cloud-connector/cloudconnector:main-47c01227a620a3dbd85b66e177205c06c0f7a52e|Running        |2023-01-31T11:15:24.981470717Z  |0             |

**Review by mikehaller — APPROVED** [review]

File: content/en/docs/build/dev-and-maintenance/shell/sdv-health.md (hunk 2 above)
Comment (vasilvas99) [reply content/en/docs/build/dev-and-maintenance/shell/sdv-health.md]: Fixed

**Review by mikehaller — APPROVED** [review]

**Pull request merged**